coderd/rbac/roles_test.go

```diff
@@ -496,6 +496,15 @@ func TestRolePermissions(t *testing.T) {
 				false: {setOtherOrg, userAdmin, templateAdmin, memberMe, orgTemplateAdmin, orgUserAdmin, orgAuditor},
 			},
 		},
+		{
+			Name:     "PrebuiltWorkspace",
+			Actions:  []policy.Action{policy.ActionRead, policy.ActionUpdate, policy.ActionDelete},
+			Resource: rbac.ResourcePrebuiltWorkspace.WithID(uuid.New()).InOrg(orgID).WithOwner(memberMe.Actor.ID),
+			AuthorizeMap: map[bool][]hasAuthSubjects{
+				true:  {owner, orgMemberMe, templateAdmin, orgTemplateAdmin},
+				false: {setOtherOrg, userAdmin, memberMe, orgAdmin, orgUserAdmin, orgAuditor},
+			},
+		},
```

Comment:
I think this needs to be two separate tests:

1) owner, templateAdmin, orgTemplateAdmin can delete and update
2) owner, templateAdmin, orgTemplateAdmin, orgMemberMe can read

Reply:
With the fix in https://github.com/coder/coder/commit/afc5359c41ff95b3af207586a2d4d67463275b2b the `orgMemberMe` role no longer has permissions on `prebuilt_workspaces`.